lib/fhir_models/bootstrap/preprocess.rb

```diff
@@ -152,6 +152,15 @@ def self.pre_process_xml_example(filename)
         f = File.open(filename, 'w:UTF-8')
         f.write(doc.to_xml)
         f.close
+
+        # Remove the weird parantheses on xml example filenames
+        # we do this so they match the names of the json examples
+        if filename.include?('(') && filename.include?(')')
```

Comment:
I really wish there was a way to remove the method call parentheses on this line, but ruby/rubocop don't like it at all. Oh well!

Reply:
Rubocop only likes terrible things.